services/validate.js

```diff
@@ -19,12 +17,19 @@ fs.readdirSync(baseDir)
   })
   .forEach(function (fileName) {
     try {
-      validator.addSchema(fileName, require(path.join(baseDir, fileName)))
+      validator.addSchema(JSON.parse(path.join(baseDir, fileName)), fileName)
```

Comment:
In combination with your [PR](https://github.com/interledgerjs/five-bells-ledger/pull/379) the ledger's tests are passing.

I ran as well the connector's unit tests against the new validator code. I fixed some failing test cases, but [one test case still has to be fixed](https://github.com/interledgerjs/ilp-connector/issues/262). Thanks for your input.